services/keybase/keybase-xlm.tester.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+const Joi = require('joi')
+
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('existing stellar address')
+  .get('/skyplabs.json')
+  .expectJSONTypes(
+    Joi.object({
+      name: 'xlm',
+      value: Joi.string(),
```

Comment:
Same as before, `Joi.string()` on its own will allow `not found`.

Reply:
I used the `withRegex(/^(?!not found$)/)` regex as I haven't found a good one for Stellar addresses. I will read the Stellar documentation to figure it out when I get a chance.